.changelog/unreleased/api-breaking/provider/xxxx-priority-validators.md

```diff
@@ -0,0 +1,2 @@
+- Allow consumer chains to specify a list of priority validators that are included in the validator set before other validators are considered
```

Comment:
Rename the file (here and below) so they do not have `xxx`. Probably add a verb there as well to make it clear from the name what this does (e.g., `introduce-priority...`).

Reply:
The files are now renamed.